service/src/main/kotlin/io/provenance/explorer/service/async/AsyncCaching.kt

```diff
@@ -322,17 +321,17 @@ class AsyncCaching(
                                 .value.toLong()
                                 .let { id ->
                                     pair.second.toMsgSubmitProposal().let {
-                                        govService.saveProposal(id, txInfo, it.proposer)
+                                        govService.saveProposal(id, txInfo, it.proposer, true)
```

Comment:
We should reference it as a named param `isSubmit = true` to give it a bit more clarity as to what the boolean value does. Also, what does `isSubmit` mean for `saveProvosal()` and can we use a default value?